plugins/modules/sonic_tacacs_server.py

```diff
@@ -123,7 +124,7 @@
       - In case of merged, the input mode configuration will be merged with the existing tacacs server configuration on the device.
       - In case of deleted the existing tacacs server mode configuration will be removed from the device.
     default: merged
```

Comment:
Please add descriptions for replaced and overridden.

Reply:
Added. Thanks!